framework/scripts/wazuh_clusterd.py

```diff
@@ -9,11 +9,22 @@
 import logging
 import os
 import signal
+import subprocess
 import sys
+from functools import partial
 
+from wazuh.core.common import WAZUH_PATH
 from wazuh.core.utils import clean_pid_files
 from wazuh.core.wlogging import WazuhLogger
 
+CLUSTER_PROCESS_NAME = 'wazuh-clusterd'
```

Comment:
Since the cluster will be the default in 5.0, we should review the script and the process naming. There is no reference to `wazuh-server`

Reply:
We decided to perform these changes in a separate issue because of the amount of code involved.